Modelica/Resources/Reference/Modelica/Mechanics/MultiBody/Examples/Constraints/PrismaticConstraint/comparisonSignals.txt

```diff
@@ -0,0 +1,11 @@
+time
+bodyOfConstraint.v_0[1]
+bodyOfConstraint.v_0[2]
+bodyOfConstraint.v_0[3]
+constraint.frame_b.r_0[1]
```

Comment:
Instead of `constraint.frame_b.r_0[1:3]` we can use `bodyOfConstraint.r_0[1:3]`.